_layouts/post.html

```diff
@@ -28,7 +28,7 @@
           </ol>
           <div class="post-title text-center">
             <img class="img-center" src="{{ page.image }}" alt="post logo">
-            <label>{{ page.title }}</label>
+            <h1>{{ page.title }}</h1>
```

Comment:
hope had already fixed this in an independent PR (still open)

Reply:
Roger. Assuming it's the same fix, whichever merges first, the other can pull it in.